constants.json

```diff
@@ -4,6 +4,7 @@
   "managementTitle": "Gravitee.io Management",
   "devMode": false,
   "userCreationEnabled": true,
+  "localLoginEnabled": true,
```

Comment:
I'd prefer to put this property in `constants.tpl.json` file.
Also, I think it's better to set this property as part of the `authentication` section (https://github.com/gravitee-io/gravitee-management-webui/blob/master/constants.tpl.json#L26).

Reply:
will do

Reply:
done